app/org/sagebionetworks/bridge/play/controllers/ConsentController.java

```diff
@@ -79,6 +79,7 @@ public Result withdrawConsent() throws Exception {
         final long withdrewOn = DateTime.now().getMillis();
         
         consentService.withdrawConsent(study, session.getUser(), withdrawal, withdrewOn);
+        updateSessionUser(session, session.getUser());
```

Comment:
Minor: As far as I can tell, consentService.withdrawConsent() sets user.consent to false, but doesn't update the sharing scope on the user session. While this has no functional impact, it does mean that the user sessions returned by our HTTP interface will incorrectly state the user's sharing scope.